cli/src/installers/dbContainer.ts

```diff
@@ -5,11 +5,24 @@ import { PKG_ROOT } from "~/consts.js";
 import { type Installer } from "~/installers/index.js";
 import { parseNameAndPath } from "~/utils/parseNameAndPath.js";
 
+// Sanitizes a project name to ensure it adheres to Docker container naming conventions.
+const sanitizeName = (name: string): string => {
+  return name
+    .replace(/[^a-zA-Z0-9_.-]/g, "_") // Replace invalid characters with underscores
+    .toLowerCase(); // Convert to lowercase for consistency
+};
+
+interface DbContainerInstallerOptions {
+  projectDir: string;
+  databaseProvider: string;
+  projectName: string;
+}
+
 export const dbContainerInstaller: Installer = ({
   projectDir,
   databaseProvider,
   projectName,
-}) => {
+}: DbContainerInstallerOptions) => {
```

Comment:
what's this type supposed to do? The function is typed as an `Installer` which we iterate over. We expect all installers to have the same signatures so this typing doesn't make much sense

```suggestion
export const dbContainerInstaller: Installer = ({
  projectDir,
  databaseProvider,
  projectName,
}) => {
```